public/firebase-messaging-sw.js

```
@@ -0,0 +1,72 @@
+importScripts("https://www.gstatic.com/firebasejs/10.11.0/firebase-app-compat.js");
+importScripts("https://www.gstatic.com/firebasejs/10.11.0/firebase-messaging-compat.js");
+
+// Firebase 설정
+const firebaseConfig = {
+  apiKey: import.meta.env.FCM_API_KEY,
+  authDomain: import.meta.env.FCM_AUTH_DOMAIN,
+  projectId: import.meta.env.FCM_PROJECT_ID,
+  storageBucket: import.meta.env.FCM_STORAGE_BUCKET,
+  messagingSenderId: import.meta.env.FCM_MESSAGING_SENDER_ID,
+  appId: import.meta.env.FCM_APP_ID,
+  measurementId: import.meta.env.FCM_MEASUREMENT_ID
```

ESLint

[error] Insert `,`

(prettier/prettier)